test/api/test_plan_serialization.cpp

```diff
@@ -43,3 +42,8 @@ TEST_CASE("Test plan serialization", "[api]") {
 	new_plan = optimizer.Optimize(move(new_plan));
 	printf("%s\n", new_plan->ToString().c_str());
 }
+
+TEST_CASE("Test plan serialization", "[api]") {
+	test_helper("SELECT last_name,COUNT(*) FROM parquet_scan('data/parquet-testing/userdata1.parquet') GROUP BY last_name");
+	test_helper("SELECT UNNEST([1, 2, 3]);"); // tests logical_dummy_scan, logical_unnest
+}
```

Comment:
Let's try to have one `TEST_CASE` per operator?
```suggestion
TEST_CASE("Basic plan serialization test", "[plan serialization]") {
	test_helper("SELECT last_name, COUNT(*) FROM parquet_scan('data/parquet-testing/userdata1.parquet') GROUP BY last_name");
}

TEST_CASE("Tests logical_dummy_scan, logical_unnest", "[plan serialization]") {
	test_helper("SELECT UNNEST([1, 2, 3]);");
}

// etc...
```

I even wonder if we want a macro for that like `TEST_SERIALIZATION(test_title, sql_statement)`?

Reply:
Yes, updated the test case file. PTAL. Thanks.